src/__tests__/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import assert from 'assert';
+import operate from '../logic/operate';
+
+describe('test operate', () => {
+  it('sum', () => {
+    assert(operate('45', '35', '+') === '80');
+  });
+
+  it('substraction', () => {
+    assert(operate('45', '35', '-') === '10');
+  });
+
+  it('multiplication', () => {
+    assert(operate('15', '5', 'x') === '75');
+  });
+});
```

Comment:
- [ ] According to the project requirement, All the functionalities & scenarios of the `operate.js` and `calculate.js` modules should be covered using Jest, kindly make sure to also add the division `÷` and module `%`  operations test 😄 💯